packages/testsuite/cypress/e2e/interface/test-configuration-interface.cy.ts

```diff
@@ -44,9 +44,4 @@ describe("TESTS: Configuration => Interface", () => {
       cy.verifyAttribute(managementEndpoint, address, anyAddress, !value);
     });
   });
-
-  it("Reset", () => {
-    cy.navigateTo(managementEndpoint, "interface;name=" + interfaceToEdit.name);
-    cy.resetForm(configurationFormId, managementEndpoint, address);
```

Comment:
Alternatively this line can be written to expect current warning, but don't know how to write that.

Reply:
Let's remove the test for now. We'll update this method in scope of #44.